entry.go

```diff
@@ -245,7 +244,8 @@ func (entry *Entry) fireHooks() {
 	defer entry.Logger.mu.Unlock()
 	err := entry.Logger.Hooks.Fire(entry.Level, entry)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Failed to fire hook: %v\n", err)
+		err := fmt.Errorf("Failed to fire hook: %v\n", err)
```

Comment:
Suggestion: pass the original error, not a wrapper string error. fmt.Errorf'ing here nukes things like pkg/errors and IIRC go 1.13's error wrapping. If it's important to differentiate hook errors from formatting errors I recommend using the go1.13 error formatter verb (https://golang.org/doc/go1.13#error_wrapping) and maybe implement a HookError and FormatterError types with Unwrap methods as per: https://golang.org/pkg/errors/#Unwrap

In either case error messages shouldn't start with capitals.

Reply:
PS: supporting go1.13 formatting may require the go1.13 implementation to be extracted to build tag protected implementations.